plover_speaker_id/lookup.py

```diff
@@ -12,17 +17,27 @@
 }
 
 
+
 class SpeakerTable:
   def __init__(self):
     # TODO: load speaker IDs from external file in Plover's config directory
-    self._speakers = DEFAULT_SPEAKERS
+    if not spkr_dict_path.exists():
+        with open(spkr_dict_path, "w") as f:
+            json.dump(DEFAULT_SPEAKERS, f)
+        self._speakers = DEFAULT_SPEAKERS
+    else:
+        with open(spkr_dict_path, "r") as f:
+            self._speakers = json.loads(f.read(), object_hook=lambda d: {int(k) if k.lstrip('-').isdigit() else k: v for k, v in d.items()})
```

Comment:
Not sure what's going on here -- are the speaker numbers converted to strings in the JSON? And why check for negative numbers?

Reply:
when I dumped the dict to file, the int keys turned to strings, and load back as strings with json.loads. So the hook turns the stringed ints back to true int keys